Cpp/odin-views/jwt.renewal.cpp

```diff
@@ -30,11 +30,11 @@ namespace {
                 fostlib::http::server::request &req,
                 const fostlib::host &host) const {
 
-            if (!req.headers().exists("__app") && !req.headers().exists("__user")) {
+            if (not req.headers().exists("__app") && not req.headers().exists("__user")) {
```

Comment:
Is it not enough to test only `__user` here? Can `__app` app ever be set without `__user`? @tleekkul

Reply:
`__app` can be set without `__user`.

Reply:
Interesting. What does that mean? App based JWT with no logged in user? Then it should go down the insecure path for the secure view?

Reply:
It means this is the app token, not user token. Maybe we want to change this behavior to be every jwt have to have `sub` value. For the app token, `sub` must refer to the app ID.

Reply:
I think my point is that this particular API can only be for refreshing the JWT for a logged in user. It cannot be used for anything else. The app JWT for when a user is not logged in cannot expire I think or we have other problems. Doesn't that affect which of these can be set?

Reply:
I get your point. Yes, it enough to test only `__user` here.

Reply:
And we don't set expiry on the app JWT do we? What's the expectation for what should happen once the log in JWT expires? Got to say I think I'm still somewhat confused about how some of this meant to work. I think we should go through on the whiteboard and then write it all down.